resources/css1.css

```diff
@@ -187,6 +199,10 @@ body{
 .background{
     margin-top: 20px;
 }
+.about_h2{
+    float: left;
+    font-size: 1.5em;
```

Comment:
try to stick to the one unit declaration for `font-size` (i see that there is mix between px and em).
[Something to Read](http://kyleschaeffer.com/development/css-font-size-em-vs-px-vs-pt-vs/)